site/_apps/zuix-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 2
+title: 'Zuix HN'
+github-title: 'genielabs/zuix-hn'
+library: 'ZUIX'
+libraries: 
+    - name: 'ZUIX'
+module-bundling: 'zuix-bundler'
+service-worker: 'Application Shell'
+perfomance-patterns: 'Lazy loaded components with caching'
+server-side-rendering: 'None'
+api: 'Hacker News Firebase API'
+hosting: 'Firebase'
+authors: 
+    - name: 'Gene'
+lighthouse: '91/100'
+interactive-em: '3.8s'
+interactive-faster-3g: '2.6s'
```

Comment:
It looks like you obtained these results from WPT` First View` values. For the rest of the implementations we've been using the times from the Lighthouse results linked through Webpagetest. You may want to do the same here and just use the `First Interactive` times linked in those reports (please let us know however if you're seeing issues getting those values)